packages/material-ui-utils/src/chainPropTypes.ts

```diff
@@ -3,13 +3,11 @@ import * as PropTypes from 'prop-types';
 export default function chainPropTypes(
   propType1: PropTypes.Validator<unknown>,
   propType2: PropTypes.Validator<unknown>,
-): PropTypes.Requireable<unknown> {
+): PropTypes.Validator<unknown> {
```

Comment:
Not sure about this one, it should create an issue on 

https://github.com/oliviertassinari/material-ui/blob/714dc1a85d15a3414e2818ec81ee31820e8aedcf/packages/material-ui-utils/src/elementAcceptingRef.js#L46-L47

but at the same time, this looks completely safe:

```jsx
var foo = () => null;
foo.isRequired = false;
```

Reply:
`PropTypes.Requireable<unknown>` means there already is a `isRequred` which is not true. We only assign that later.